packages/client/src/components/Modal/__tests__/modal.test.tsx

```diff
@@ -0,0 +1,55 @@
+import { fireEvent, render, screen } from '@testing-library/react';
+
+import { Modal } from '../Modal';
+
+const onClose = jest.fn();
+
+describe('Modal snapshots', () => {
+  test('should render modal', () => {
+    const tree = render(
+      <Modal isOpen={true} onClose={onClose}>
```

Comment:
true не обязательно передать
```suggestion
      <Modal isOpen onClose={onClose}>
```